.github/workflows/test.yml

```diff
@@ -2,13 +2,68 @@ name: Test
 
 on:
   push:
-  pull_request:
-    branches:
-      - main
 
 jobs:
+  inch:
+    name: Analyse Documentation
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        elixir:
+        - "1.11"
+        otp:
+        - "23.0"
```

Comment:
If there's only one option in the matrix, maybe we can simplify by removing the matrix strategy altogether?

Reply:
I found it easier to use this matrix and the references (e.g. `{{ matrix.elixir }}` instead of hard coding the versions in cache keys, etc. It'll make future updates to these versions easier as well.

Reply:
Sure! That works too.